### DISCUSSION TITLE
Issue when trying to run server with react native

### DISCUSSION THREAD

**Discussion by samMcCarthyANDDigital** [mswjs/msw#1225] [Q&A] [answered]
Hi

I've seeing the following issue when trying to run msw with react native: 
![image](https://user-images.githubusercontent.com/96571822/166485097-14e302be-d3df-4030-853c-e0f9683d8106.png)

I'm running the following versions:
msw: 0.39.2
node: 18.10.0
react native:  0.67.3

I followed the setup guide for the node runner with graphql

**Comment by kettanaito** [comment, marked as answer]
Hi, @samMcCarthyANDDigital. Thanks for reporting this. 

This will be fixed in the next major release as we've migrated away from the `debug` dependency.
